js/index.js

```diff
@@ -0,0 +1,147 @@
+const btnMenu = document.querySelector('.navbar-toggler');
+const iconBtn = document.querySelector('#icon-btn');
+
+if (btnMenu) {
+  btnMenu.addEventListener('click', () => {
+    iconBtn.classList.toggle('fa-xmark');
+  });
+}
+const featureCooks = document.querySelector('#features-cook');
+const featuresData = [
+  {
+    name: 'Massimo Bottura',
+    job: 'Chef Patron, Osteria Francescana (Modena, Italy)',
+    description: `Massimo Bottura is an Italian chef who owns and runs the three-Michelin-starred Osteria Francescana, located in Modena, Italy.
+    `,
+    image: 'images/Massimo Bottura.webp',
+  },
+  {
+    name: 'Dominique Crenn',
+    job: 'Chef/Owner, Atelier Crenn (San Francisco, USA)',
+    description: `
+    Dominique Crenn is a French-American chef and restaurateur, and the first female chef in the US to earn three Michelin stars.`,
+    image: 'images/Dominique Crenn.jpg',
+  },
+  {
+    name: 'Yotam Ottolenghi',
+    job: 'Chef/Owner, Ottolenghi (London, UK)',
+    description: `Yotam Ottolenghi is  cookbook author, and restaurateur, and the owner of several highly successful restaurants in London.
+    `,
+    image: 'images/Yotam Ottolenghi.jpg',
+  },
+  {
+    name: 'Alex Atala',
+    job: 'Chef/Owner, D.O.M. (Sao Paulo, Brazil)',
+    description: `Alex Atala is a highly acclaimed restaurant in Sao Paulo that specializes in contemporary Brazilian cuisine. 
+    `,
+    image: 'images/Alex Atala.jpeg',
+  },
+];
+
+const createImage = (className, src, alt) => {
+  const img = document.createElement('img');
+  img.className = className;
+  img.src = src;
+  img.alt = alt;
+  return img;
+};
+
+const createDiv = (className) => {
+  const div = document.createElement('div');
+  div.className = className;
+  return div;
+};
+
+const createtext = (textData, tagElement) => {
+  const tag = document.createElement(tagElement);
+  const text = document.createTextNode(textData);
+  tag.appendChild(text);
+  return tag;
+};
+
+const createFeatures = (feature) => {
+  const columnDiv = createDiv('col-md-6 d-flex mt-5 cook-person');
+
```

Comment:
**[OPTIONAL]** Currently, the image is overflowing over the text please consider resolving the issue to meet the design guidelines ☑️ 

![image](https://user-images.githubusercontent.com/94223367/233468709-6c4362fb-d193-427c-9cc1-7a2f0f01c386.png)